api/docs/installation/heroku.md

```diff
@@ -77,23 +57,26 @@ Addons and Environment Variables:
 Now it's time to add the addons and the environment variables to your app at the Heroku server. For the app I mentioned in the beginning, I type:
 
 ````
-$ heroku addons:add heroku-postgresql:dev
-$ heroku pg:promote HEROKU_POSTGRESQL_ONYX_URL
+$ heroku addons:create heroku-postgresql:hobby-dev
+````
+
+When you run this command the output will a url which you should note as HEROKU_POSTGRESQL_URL
+
+````
+$ heroku pg:promote HEROKU_POSTGRESQL_URL
 $ heroku config:set MAIL_USERNAME="<login>"
 $ heroku config:set MAIL_PASSWORD="<password>"
 ````
 
-Adding Requirements :
-
-Heroku needs to know what libraries and packages it needs to install to be able to run your application. For this, create a file requirements.txt in the root of your app, with all the libraries from your environment.
+## Deploying!
 
-``$ cat pip freeze >> requirements.txt``
+If everything is good, it's time to deploy your application.
 
-### Creation of Procfile :
+Run this command :
 
-Create a file named Procfile. The content should be :
+``$ git push heroku master``
```

Comment:
You sure, We don't need this `Proc File`, I don't think deployment will actually work without the Proc

Reply:
There is a proc file already in the repo, no need to create a new one